package.json

```diff
@@ -25,14 +27,22 @@
     "eslint": "8.25.0",
     "eslint-config-airbnb": "^19.0.4",
     "eslint-config-next": "12.3.1",
-    "jest": "^29.1.2",
+    "husky": "^8.0.1",
     "jest-environment-jsdom": "^29.1.2",
+    "jest": "^29.1.2",
+    "lint-staged": "^13.0.3",
     "npm-run-all": "^4.1.5",
+    "prettier": "2.6.2",
     "sass": "^1.55.0",
     "stylelint": "^14.13.0",
     "stylelint-config-standard": "^28.0.0",
     "stylelint-config-standard-scss": "^5.0.0"
   },
+  "lint-staged": {
+    "*.{ts,tsx,js,jsx,css,md,json}": [
+      "prettier --write"
```

Comment:
Just a note, also using `eslint --fix` here can also help to print the error and auto-fix some minor issues. It may be worth digging into.

Reply:
Found some documentation that should help with this, since Next uses its own flavor of eslint: https://nextjs.org/docs/basic-features/eslint#lint-staged

Reply:
Thanks I'll add that!